src/pages/ErrorPage/Error400.jsx

```diff
@@ -0,0 +1,10 @@
+import React from 'react';
+import PortalLayout from '../../layouts/PortalLayout';
+
+const Error400 = () => (
```

Comment:
I think the current 404 page is more suited for internal usage. e.g. invalid URL then it displays "page not found".
Failed API calls should have a separate display message - "error retrieving information" or "something went wrong" kind of message instead of "page not found"